### ISSUE TITLE
Doc-Properties auto-update macro does not support write-protected word files

### ISSUE THREAD

**Issue by jone** [4teamwork/opengever.core#3377]
Given I have the macro installed from the [documentation](https://docs.onegovgever.ch/admin-manual/docproperties/autoupdate/) on a Windows maschine, when I open a write-protected word file, there is an error because `ActiveDocument.StoryRanges` is not allowed to be executed on in write-protected word documents.

A possible solution is to catch all errors:
```vb
Sub AutoOpen()
'
' UpdateDocprops Makro
' http://www.gmayor.com/insta...
'
'
On Error GoTo ErrHandler
Dim oStory As Range
For Each oStory In ActiveDocument.StoryRanges
  oStory.Fields.Update
  If oStory.StoryType <> wdMainTextStory Then
    While Not (oStory.NextStoryRange Is Nothing)
      Set oStory = oStory.NextStoryRange
      oStory.Fields.Update
    Wend
  End If
Next oStory
Set oStory = Nothing
Exit Sub
ErrHandler:
Exit Sub
End Sub
```

I'm not sure though if this is robust enough.

The documentation should be updated with a macro which is robust enough to not crash when opening write-protected documents.

I couldn't reproduce this problem on my Mac, thus no screenshot of the error.

**Comment by lukasgraf** [comment]
Related: #3263

**Comment by phgross** [comment]
Will be solved via #3263
